core/src/main/java/jenkins/model/logging/impl/CloseableStreamBuildListener.java

```diff
@@ -0,0 +1,30 @@
+package jenkins.model.logging.impl;
+
+import hudson.model.StreamBuildListener;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.Beta;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+
+/**
+ * @author Oleg Nenashev
+ * @since TODO
+ */
+@Restricted(Beta.class)
+public class CloseableStreamBuildListener extends StreamBuildListener implements Closeable {
```

Comment:
Huh? `StreamTaskListener` is already `Closeable`.

Reply:
Yes, to be deleted at all